PlayRho/Dynamics/World.cpp

```diff
@@ -2111,6 +2111,7 @@ ContactCounter World::FindNewContacts()
     // to eliminate any node pairs that have the same body here before the key pairs are
     // sorted.
     for_each(cbegin(m_proxies), cend(m_proxies), [&](ProxyId pid) {
+        if (pid == DynamicTree::GetInvalidSize()) return;
```

Comment:
CI issues appear to be better now and I'm getting back to reviewing other pull requests like this one. Thanks for your patience!

I agree this issue is a bug that needs to be fixed and appreciate your having pointed it out. While I believe that this conditional statement will protect against it, it's concerning to me to add this here since this method is lower level code and more performance critical.

I wonder if there's way to remove the invalid sized `pid` from the `m_proxies` container before this code is even reached. I suspect that'd be better if that way only occurred when the `m_proxies` element was invalidated. If that's possible, then I'd also like an assert statement here along the lines of something like `assert( pid != DynamicTree::GetInvalidSize());`.